src/commands/feed.ts

```diff
@@ -0,0 +1,84 @@
+import { BrandingColors } from '#lib/common/constants';
+import { LanguageKeys } from '#lib/i18n/LanguageKeys';
+import { randomEnum } from '#lib/utilities/utils';
+import { EmbedBuilder, userMention } from '@discordjs/builders';
+import { Time } from '@sapphire/duration';
+import { Command, RegisterCommand } from '@skyra/http-framework';
+import { applyLocalizedBuilder, createSelectMenuChoiceName, getSupportedLanguageT, resolveKey } from '@skyra/http-framework-i18n';
+import { Json, isAbortError, safeTimedFetch, type FetchError } from '@skyra/safe-fetch';
+
+const Root = LanguageKeys.Commands.Feed;
+const FallbackImageUrl = 'https://foodish-api.com/images/burger/burger82.jpg';
+
+@RegisterCommand((builder) =>
+	applyLocalizedBuilder(builder, Root.RootName, Root.RootDescription)
+		.addUserOption((option) => applyLocalizedBuilder(option, Root.OptionsTarget).setRequired(true))
+		.addStringOption((option) =>
+			applyLocalizedBuilder(option, Root.OptionsType).addChoices(
+				createSelectMenuChoiceName(Root.PizzaKey, { value: FeedType.Pizza }),
+				createSelectMenuChoiceName(Root.BurgerKey, { value: FeedType.Burger }),
+				createSelectMenuChoiceName(Root.RizeKey, { value: FeedType.Rize }),
+				createSelectMenuChoiceName(Root.DessertKey, { value: FeedType.Dessert }),
+				createSelectMenuChoiceName(Root.DosaKey, { value: FeedType.Dosa }),
+				createSelectMenuChoiceName(Root.PastaKey, { value: FeedType.Pasta }),
+				createSelectMenuChoiceName(Root.SamosaKey, { value: FeedType.Samosa }),
+				createSelectMenuChoiceName(Root.BiryaniKey, { value: FeedType.Biryani })
+			)
+		)
+)
+export class UserCommand extends Command {
+	public override async chatInputRun(interaction: Command.ChatInputInteraction, options: Options) {
+		const type = options.type ?? randomEnum(FeedType);
+
+		const url = new URL(`https://foodish-api.com/api/images/${options.type}`);
```

Comment:
You forgot this line:
```suggestion
		const url = new URL(`https://foodish-api.com/api/images/${type}`);
```